### PR TITLE
Update instructions, position and github buttons

### LINKED ISSUE #248: Update microk8s.io homepage for V2 launch
Please update index.html - "microk8s - Zero-ops Kubernetes for workstations and edge / IoT" from the [copy doc](https://docs.google.com/document/d/1ObWmhRc4GCIhrBi8gT7yIIs0J_dn_8ravfLtcwhr3uQ/edit?usp=drivesdk)

- and move things around
    - github buttons
    - instructions

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- updated the copy and move things around
    - github buttons
    - instructions

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- compare to the [copy doc](https://docs.google.com/document/d/1ObWmhRc4GCIhrBi8gT7yIIs0J_dn_8ravfLtcwhr3uQ/edit?usp=drivesdk)

## Issue / Card

Fixes #248

## Screenshots

image

### CODE REVIEW

**Comment by anthonydillon** [comment]
@pmahnke why has the lock file changes so much? There are no dep changes?

**Comment by anthonydillon** [comment]
## QA issues
- The github buttons are visible both in the hero and menu on small screens:
image
- I think you need to snap to the mobile nav earlier:
image

File: _layouts/base.html [hunk 1]
```diff
@@ -91,6 +91,9 @@
             <a class="p-link--external" href="https://github.com/ubuntu/microk8s/issues">Bugs</a>
           </li>
         </ul>
+        <div style="margin-top: 0.7rem; margin-right: 2rem;">
```
Comment (anthonydillon) [inline _layouts/base.html:94]: Can you make this a pattern so it can be reused on other sites?

File: index.html [hunk 2]
```diff
@@ -6,19 +6,15 @@
     <div class="row u-vertically-center u-equal-height">
       <div class="col-8">
         <h1>
-          Autonomous low-ops Kubernetes for clusters, workstations, edge and IoT
+          Autonomous low-ops Kubernetes for clusters, workstations, edge and&nbsp;IoT
         </h1>
 
         <p class="p-heading--four u-sv3">
           Super-simple production-grade upstream K8s. One-command install on 42 flavours of Linux. Made for developers and devops. Great for edge and IoT.
```
Comment (anthonydillon) [inline index.html:13]: This is different in the copy doc.

File: index.html [hunk 3]
```diff
@@ -68,372 +175,217 @@ <h1>
...
+      <div class="col-8">
+        <h2>
+          Lightweight Kubernetes done right
+        </h2>
 
-    <section class="p-strip--light">
-      <div class="u-fixed-width">
-        <h2>Reliable, fast, small, upstream.</h2>
+        <p>
+          The smallest, fastest, fully-conformant upstream Kubernetes that makes clustering trivial. A single package for multiple flavours of Linux means MicroK8s is great for offline development, prototyping, and testing. Use it on a VM as a small, free, reliable K8s for CI/CD. Strict confinement and automatic updates make MicroK8s best Kubernetes for appliances. Develop IoT apps for K8s and deploy them to MicroK8s on your Linux boxes.
```
Comment (anthonydillon) [inline index.html:210]: The copy doc starts with "MicroK8s is 100% conformant upstream.."

File: index.html [hunk 4]
```diff
@@ -68,372 +175,217 @@ <h1>
...
-                <a href="https://github.com/ubuntu/microk8s/releases/download/installer-v1.0.0/microk8s-installer.exe" class="p-button--positive">
-                  <span class="p-link--external">Download MicroK8s for Windows</span>
-                </a>
-              </p>
-            </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">Featureful</h3>
+            <div class="p-matrix__desc">
+              <p>Cool things you probably want to try on a small, standard K8s are all built-in. Just enable them and go.</p>
```
Comment (anthonydillon) [inline index.html:289]: This is different from the copy doc

File: index.html [hunk 5]
```diff
@@ -28,7 +24,118 @@ <h1>
...
+            <p class="p-stepped-list__content u-stepped-list-margin">
+              <a href="https://github.com/ubuntu/microk8s/releases/download/installer-v1.0.0/microk8s-installer.exe" class="p-button--positive">
+                <span class="p-link--external">Download MicroK8s for Windows</span>
+              </a>
+            </p>
+          </li>
+
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title">Run the Installer</h3>
+            <img src="https://assets.ubuntu.com/v1/141d9f8b-winmk8s-01.png">
```
Comment (anthonydillon) [inline index.html:88]: Image is missing an `alt`

File: index.html [hunk 6]
```diff
@@ -28,7 +24,118 @@ <h1>
...
+          </li>
+
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title">Open a command line</h3>
+
+            <img src="https://assets.ubuntu.com/v1/a5fe14a5-winmk8s-04.png">
```
Comment (anthonydillon) [inline index.html:94]: Image is missing `alt`

File: _sass/_settings.scss [hunk 7]
```diff
@@ -8,3 +8,6 @@ $color-navigation-active-bar: $ubuntu-orange;
 // Sidebar
 $content-padding: 0 5%;
 $sidebar-width: 18rem;
+
+// Other
+$breakpoint-medium:    875px
```
Comment (anthonydillon) [inline _sass/_settings.scss:13]: You should use `$breakpoint-navigation-threshold` to change the menu breakpoint.

**Comment by tbille** [comment]
@pmahnke is this PR still relevant?

**Comment by pmahnke** [comment]
closing as out of date.

**Pull request closed without merging**